app/main.py

```diff
@@ -2,0 +1,23 @@
+import os.path
+import shutil
+
+
+def move_file(command: str) -> None:
+    parsed_command = command.split()
+    command_name = parsed_command[0]
+    source_file = parsed_command[1]
+    destination_file = parsed_command[2]
```

Comment:
Code Style: The variable names `command_name`, `source_file`, and `destination_file` are not very descriptive. Consider renaming them to `command_type`, `source_path`, and `destination_path` respectively. This would make your code more readable.